packages/webcomponentsjs/ts_src/platform/child-node/after.ts

```diff
@@ -10,15 +10,16 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 
 export {};
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 type Constructor<T> = new (...args: Array<any>) => T;
 
 const nativeInsertBefore = Node.prototype.insertBefore;
 const nativeGetParentNode =
-    Object.getOwnPropertyDescriptor(Node.prototype, 'parentNode')?.get! ??
+    Object.getOwnPropertyDescriptor(Node.prototype, 'parentNode')?.get ??
```

Comment:
Do these actually compile without the `!`s? I'm pretty sure I had to put them in because TS knows it's not guaranteed that some arbitrary descriptor will have a `get`.

Reply:
If you want the `!`, then you need to remove the previous `?`.
Because can't ensure a not-null after a posible undefined value.

Reply:
Oh interesting, I had always assumed the `!` was applying to specifically the `.get` part, but it sounds like it actually applies to the entire expression leading up to that point.

Reply:
Exactly!